testsuite/integration-tests/src/test/java/org/jboss/resteasy/test/providers/sse/SseTest.java

```diff
@@ -494,6 +494,24 @@ public void testNoReconnectAfterEventSinkClose() throws Exception
          // falls through
          e.printStackTrace();
       }
+     
+      Assert.assertEquals("Received unexpected events", "[thing1, thing2, thing3]", results.toString());
+      //test for [Resteasy-1863]:SseEventSourceImpl should not close Client instance
+      results.clear();
+      WebTarget target2 = client.target(generateURL("/service/server-sent-events/closeAfterSent"));
+      SseEventSourceImpl sourceImpl2 = (SseEventSourceImpl)SseEventSource.target(target2).build();
+      sourceImpl2.setAlwasyReconnect(false);
+      try (SseEventSource source = sourceImpl2)
+      {
+         source.register(event -> results.add(event.readData()));
+         source.open();
+         Thread.sleep(1000);
+      }
+      catch (InterruptedException e)
+      {
+         // falls through
+         e.printStackTrace();

```

Comment:
Use logger pls

and maybe also do the switch of e.printStackTrace(); on line 495

Reply:
@rsvoboda  I'll fix this.  Thanks.